spatial-tracking-explainer.md

```diff
@@ -281,9 +281,9 @@ function onSessionStarted(session) {
   xrSession.requestReferenceSpace('unbounded').then((referenceSpace) => {
     xrReferenceSpace = referenceSpace;
   }).catch(() => {
-    // If an unbounded reference space is not available, request an eye-level
```

Comment:
It's totally minor, but I feel like the "local-floor" and "bounded-floor" would make a better example here.